docs/source/apis/Products.rst

```diff
@@ -0,0 +1,412 @@
+   .. automethod:: get_lowest_priced_offers_for_asin
+
+      .. rubric:: Example
+      .. code-block:: python
+
+          resp = products_api.get_lowest_priced_offers_for_asin(
+              marketplace_id=my_market,
+              asins=["B085G58KWT"],
+              condition="New" # Any, New, Used, Collectible, Refurbished, Club. Default = Any
+          )
+
+   .. ############################################### GetMyFeesEstimate ################################################
+   .. automethod:: get_my_fees_estimate
+
+      Accepts one or more :py:class:`FeesEstimateRequest <mws.models.products.FeesEstimateRequest>` instances as
+      arguments:
+
+      .. rubric:: Example
+      .. code-block:: python
+
+          estimate_request = FeesEstimateRequest(...)
+          resp = products_api.get_my_fees_estimate(estimate_request)
+
+      Multiple estimates can be requested at the same time, as well:
+
+      .. code-block:: python
+
+          estimate_request1 = FeesEstimateRequest(...)
+          estimate_request2 = FeesEstimateRequest(...)
+          resp = products_api.get_my_fees_estimate(estimate_request1, estimate_request2, ...)
+
+   .. ################################################ GetMyPriceForSKU ################################################
+   .. automethod:: get_my_price_for_sku
+
+      .. rubric:: Example
+      .. code-block:: python
+
+          resp = products_api.get_my_price_for_sku(
+            marketplace_id = my_market,
+            skus="OO-NL0F-795Z",
+            condition="New"
+            # Any, New, Used, Collectible, Refurbished, Club. Default = All
+        )
+
+   .. ################################################ GetMyPriceForASIN ###############################################
+   .. automethod:: get_my_price_for_asin
+
+      .. rubric:: Example
+      .. code-block:: python
+
+          resp = products_api.get_my_price_for_asin(
+              marketplace_id=my_market,
+              asins="B07QR73T66",
+              condition="New"
+              # Any, New, Used, Collectible, Refurbished, Club. Default = All
+          )
+
+   .. ########################################### GetProductCategoriesForSKU ###########################################
+   .. automethod:: get_product_categories_for_sku
+
+      .. rubric:: Example
+      .. code-block:: python
+
+          resp = products_api.get_product_categories_for_sku(
+              marketplace_id=my_market,
+              sku="OO-NL0F-795Z"
+          )
+
+   .. ########################################### GetProductCategoriesForASIN ##########################################
+   .. automethod:: get_product_categories_for_asin
+
+      .. rubric:: Example
+      .. code-block:: python
+
+          resp = products_api.get_product_categories_for_asin(
+              marketplace_id=my_market,
+              asin="B07QR73T66"
+          )
+
+Data models
+===========
```

Comment:
The Data models are not part of the API module itself, but they are relevant to the API and should be outlined in the same doc for completeness. The same is true in the InboundShipments documentation (which was added in the Oct-Dec 2020 sprint).